appinventor/components/src/com/google/appinventor/components/runtime/AppInventorCompatActivity.java

```diff
@@ -7,6 +7,7 @@
 
 import android.app.Activity;
 import android.content.res.Configuration;
+import android.content.res.Resources;
```

Comment:
Most of the logic additions in this file should be pushed down into the Form component. This activity is used for other components as well (ListPicker activity, FullScreenVideo, possibly extensions), so we want the logic shared with those other activities.